prepare-chroot-debian

```diff
@@ -186,11 +186,11 @@ add_repo "deb-src https://deb.debian.org/debian ${DIST} main"
 del_repo "//deb.qubes-os.org/"
 
 if [ -n "$USE_QUBES_REPO_VERSION" ]; then
-    add_repo "deb [arch=amd64] https://deb.qubes-os.org/r${USE_QUBES_REPO_VERSION}/vm $DIST main"
+    add_repo "deb [arch=amd64 signed-by=/usr/share/keyrings/qubes-archive-keyring.gpg] https://deb.qubes-os.org/r${USE_QUBES_REPO_VERSION}/vm $DIST main"
     if [ "0$USE_QUBES_REPO_TESTING" -gt 0 ]; then
-        add_repo "deb [arch=amd64] https://deb.qubes-os.org/r${USE_QUBES_REPO_VERSION}/vm ${DIST}-testing main"
+        add_repo "deb [arch=amd64 signed-by=/usr/share/keyrings/qubes-archive-keyring.gpg] https://deb.qubes-os.org/r${USE_QUBES_REPO_VERSION}/vm ${DIST}-testing main"
     fi
-    chroot $DIR apt-key add - < ${DEBIAN_PLUGIN_DIR}keys/qubes-debian-r${USE_QUBES_REPO_VERSION}.asc
+    cp ${DEBIAN_PLUGIN_DIR}keys/qubes-debian-r${USE_QUBES_REPO_VERSION}.asc "${DIR}/usr/share/keyrings/qubes-archive-keyring.gpg"
```

Comment:
apt doesn't like armor key format, this needs `gpg --dearmor`. I can add it as a follow-up commit.

Reply:
Armor key format supported. Just keep the file ending `.asc`. Certainty: high

For example, Kicksecure, Whonix are using `/usr/share/keyrings/derivative.asc` for a few Debian releases now and this works great.

So the untested line that would probably work is:

    cp ${DEBIAN_PLUGIN_DIR}keys/qubes-debian-r${USE_QUBES_REPO_VERSION}.asc "${DIR}/usr/share/keyrings/qubes-archive-keyring.asc"

Reply:
Just tested, indeed it works, thanks!